src/payment/amazon-pay.md

```diff
@@ -1,27 +1,36 @@
 ---
 title: Amazon Pay
+redirect_from:
+  - /payment/amazon-pay-resources.html
 ---
 
-
-Amazon Pay and Login with Amazon offer your customers a secure and convenient way to pay that is easy to set up, and that helps to increase conversions and sales. Customers who log in to your store with their Amazon credentials can speed through checkout using the payment and shipping information that is already stored in their Amazon accounts.
+Amazon Pay and _Login with Amazon_ offer your customers a secure and convenient way to pay that is easy to set up, and that helps to increase conversions and sales. Customers who log in to your store with their Amazon credentials can speed through checkout using the payment and shipping information that is already stored in their Amazon accounts.
```

Comment:
why is one ital and the other not?

Reply:
Was in quotes. I looked at Amazon docs and I think better to not italicize and just do it the way that they do.